index.js

```diff
@@ -420,8 +420,31 @@ module.exports = {
 			'error',
 			'never',
 		],
-		// Disabled because of https://github.com/xojs/eslint-config-xo/issues/27
-		// 'object-property-newline': 'error',
+		'object-curly-newline': [
+			'error',
+			{
+				ObjectExpression: {
+					multiline: true,
+					minProperties: 2,
+					consistent: true,
+				},
+				ObjectPattern: {
+					multiline: true,
+					minProperties: 4,
```

Comment:
What's the reasoning?

Reply:
Thank you for asking! Upon reflection, I believe this might not be a good idea. I am leaning towards to set this option as never now until the community enhance the `object-property-newline` rule or add `destructuring-property-newline`.

Happy to hear your thoughts :)

Reply:
If we add `object-property-newline ` as well we can achieve the following result

Incorrect
```js

let a = {foo: 1
};
let b = {
    foo: 1};
let c = {foo: 1, bar: 2
};

Object.assign(stream, {isTTY: true, columns, rows});

```

Correct
```js
Object.assign(stream, {
    isTTY: true,
    columns,
    rows
});


let a = {foo: 1};
let b = {foo: 1};
let c = {
    foo: 1,
    bar: 2
};
```